### ISSUE TITLE
Indicate unreachable code

### ISSUE THREAD

**Issue by Darkyenus** [Darkyenus/glsl4idea#18]
That is, code after return/break/continue/discard should be striked-out or something

**Comment by Darkyenus** [comment]
That seems to work.

**Comment by AbigailBuccaneer** [comment]
```C
int x;

void main() {
    if (x > 1) return; else x = 2; // the 'else' keyword and case are marked as unreachable
    if (x > 1) { return; } else { x = 2; } // this is annotated correctly 
    { return; x = 0; } // the assignment expression will be marked as unreachable...
    x *= 2; // ...but the rest of the function won't be
}
```

**Comment by Darkyenus** [comment]
Hmm, you are right, it is very basic. Those examples should be possible to fix sort of easily, and maybe it can be extended on after completing flow analysis, but even IntelliJ Java does not do that, so it should be fine.

**Comment by AbigailBuccaneer** [comment]
Fixed in #29 :)
